golang/pkg/dagent/dagent.go

```diff
@@ -39,7 +39,7 @@ func Serve(cfg *config.Configuration) {
 	grpc.Init(grpcContext, grpcParams, &cfg.CommonConfiguration, grpc.WorkerFunctions{
 		Deploy:     utils.DeployImage,
 		Watch:      utils.GetContainersByNameCrux,
-		Delete:     utils.DeleteContainerByName,
+		Delete:     utils.DeleteContainerByNameGrpc,
```

Comment:
This suffix is weird, these are all grpc functions, could this be solved in another way?

Reply:
We talked about this in person, fixed.